crates/goose-cli/src/commands/session.rs

```diff
@@ -41,34 +61,26 @@ pub async fn build_session<'a>(
         );
     }
 
-    let loaded_profile = load_profile(profile);
-
-    // Set environment variables for provider configuration
-    set_provider_env_vars(&loaded_profile.provider, &loaded_profile);
+    let config = Config::load().unwrap_or_else(|_| {
+        println!("No configuration found. Please run 'goose configure' first.");
+        process::exit(1);
+    });
```

Comment:
Do we want to distinguish between "no config" vs "invalid" config?
I made my config file invalid yaml and just got
```
No configuration found. Please run 'goose configure' first.
```